packages/stats/src/Monitor.ts

```
@@ -61,10 +63,10 @@ export default class Monitor {
   private container: HTMLElement;
   private readonly items: string[];
 
-  constructor(gl: WebGLRenderingContext | WebGL2RenderingContext) {
-    this.core = new Core(gl);
+  constructor(engine: Engine) {
+    this.core = new Core(engine);
     this.items = [];
-    this.items = ["fps", "memory", "drawCall", "triangles", "textures", "shaders", "size", "webglContext"];
+    this.items = ["fps", "memory", "totalGraphicsMemory", "textureMemory", "bufferMemory", "drawCall", "triangles", "webglContext"];
```

ESLint

[error] Replace `"fps",·"memory",·"totalGraphicsMemory",·"textureMemory",·"bufferMemory",·"drawCall",·"triangles",·"webglContext"` with `⏎······"fps",⏎······"memory",⏎······"totalGraphicsMemory",⏎······"textureMemory",⏎······"bufferMemory",⏎······"drawCall",⏎······"triangles",⏎······"webglContext"⏎····`

(prettier/prettier)